src/python/pants/backend/jvm/tasks/junit_run.py

```diff
@@ -160,7 +163,12 @@ def register_options(cls, register):
                   'All tests output also redirected to files in .pants.d/test/junit.')
     register('--cwd', advanced=True, fingerprint=True,
              help='Set the working directory. If no argument is passed, use the build root. '
-                  'If cwd is set on a target, it will supersede this argument.')
+                  'If cwd is set on a target, it will supersede this argument. It is an error to '
+                  'use this option in combination with `--chroot`')
+    register('--chroot', advanced=True, fingerprint=True, type=bool, default=False,
+             help='Run tests in a chroot. Any loose files tests depend on via `{}` dependencies '
+                  'will be copied to the chroot. It is an error to use this option in combination '
+                  'with `--cwd`'.format(Files.alias()))
```

Comment:
Now called out and tested. Yes, adding the equivalent of a pytest `--fast`/`--no-fast` to allow opt-in to per-target is a next step in a separate PR.